build.gradle

```diff
@@ -226,8 +229,9 @@ allprojects {
     ]
 
     options.errorprone {
-      excludedPaths = '.*/(generated/*.*|.*ReferenceTest_.*|build/.*/annotation-output/.*)'
-
+      //excludedPaths = '.*/(generated/*.*|.*ReferenceTest_.*|build/.*/annotation-output/.*)'
```

Comment:
what's the consequence of this change - do we end up flagging issues within ref tests?

Reply:
@macfarla  Should have removed the commented line. I think the option `disableWarningsInGeneratedCode = true` has started ignoring quite a bit of files. Also, the `ReferenceTest_` tests are generated in the `generated` sub-folder. In summary, we are still good by updating the excludedPath to only ignore `generated` subfolders.